test/specs/elements/List/List-test.js

```diff
@@ -37,12 +37,14 @@ describe('List', () => {
       const wrapper = shallow(<List />)
 
       wrapper.find('ListItem').should.have.lengthOf(0)
+      wrapper.should.have.prop('role', 'list')
```

Comment:
Would prefer we add separate `it()` blocks for the accessibility assertions:

```jsx
it('is accessible', () => {...})
// or 
it('has proper aria props', () => {...})
```

Something to this effect.  The idea is to keep the `it()` strings descriptive to exactly what is being asserted inside of them.